[Editor Fonts] Add Noto Sans Bold font variant for supported languages.

bruvzg opened the pull request:
Adds bold variants of Arabic, Bengali, Devanagari, Georgian, Hebrew, Malayalam, Oriya, Sinhala, Tamil, Telugu and Thai fonts for more consistent editor interface (many editor elements are using bold font since #49148).
Currently, bold font are available only for the main Noto Sans font (Latin, Cyrillic and Greek).

CJK font (composed of Droid Sans Japanese and Droid Sans Fallback, ~5 MB in size) do not have bold variant and is left intact (Noto Sans CJK fonts are about 15 MB for each variant).

Adds about a megabyte to the editor binary size.

<img width="364" alt="_bold_fonts" src="https://user-images.githubusercontent.com/7645683/126129213-b102874e-f637-41d1-b080-808d35f59d8d.png">

bruvzg committed:
[Editor Fonts] Add Noto Sans Bold font variant for supported languages.

akien-mga commented:
Looks like this also updates existing fonts, did you sync with a newer version of the fonts? If so the relevant section in `thirdparty/README.md` should be updated too (as well as `s/NotoNaskhArabicUI_Regular.ttf/NotoNaskhArabicUI*.ttf/`).

bruvzg commented:
> Looks like this also updates existing fonts, did you sync with a newer version of the fonts? If so the relevant section in thirdparty/README.md should be updated too (as well as s/NotoNaskhArabicUI_Regular.ttf/NotoNaskhArabicUI*.ttf/).

Yes, all fonts are taken from current download from https://www.google.com/get/noto/
I have updated README, and also updated main Noto font (it's no longer have UI suffix, but it's the same font with the same metrics).

akien-mga commented:
Thanks!
